### PR TITLE
Z723

### PR DESCRIPTION
**Pull request by jansaidl**

### CODE REVIEW

File: src/cliAction/login/handler.go [hunk 1]
```diff
@@ -140,16 +140,16 @@ func (h *Handler) loginWithPassword(_ context.Context, login, password string) e
 
 	data := h.storage.Data()
 	data.Token = tokenResponseObject.Token
```
Comment (kawaiiyoshiyuki) [inline src/cliAction/login/handler.go:142]: I might miss something but do we need the same code here and on line 144? 🤔

**Review by kawaiiyoshiyuki — COMMENTED** [review]
I am afraid the code is behind my knowledge but we need to update the tests in  handler_test.go :)

File: src/cliAction/login/handler.go (hunk 1 above)
Comment (jansaidl) [reply src/cliAction/login/handler.go]: Yes, we don't.

**Review by kawaiiyoshiyuki — APPROVED** [review]
looks good now :)

**Pull request merged**